### PR TITLE
BB-127 Add replayProcessor probeServer logic

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Comment by nicolas2bert** [comment]
ping

File: extensions/replication/replayProcessor/task.js [hunk 1]
```diff
@@ -48,22 +48,23 @@ const queueProcessor = new QueueProcessor(
...
  * @returns {ProbeServerConfig|undefined} Config for site or undefined if not found
  */
-function getProbeConfig(queueProcessorConfig, site) {
-    return queueProcessorConfig &&
-        queueProcessorConfig.probeServer &&
-        queueProcessorConfig.probeServer.filter(c => c.site === site)[0];
+function getProbeConfig(replayProcessorConfig, site, topicName) {
+    return replayProcessorConfig &&
+        replayProcessorConfig.probeServer &&
+        replayProcessorConfig.probeServer.find(c => c.site === site && c.topicName === topicName);
```
Comment (miniscruff) [inline extensions/replication/replayProcessor/task.js:62]: Now that we have node 16 we can use optional chaining here if you want.
```suggestion
    return replayProcessorConfig?.probeServer?.find(c => c.site === site && c.topicName === topicName);
```

Comment (nicolas2bert) [reply extensions/replication/replayProcessor/task.js:62]: That would require some linter change. Let's handle it separately.

**Review by miniscruff — APPROVED** [review]

**Review by naren-rajendran — APPROVED** [review]

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**